### PR TITLE
BrenKt-JEemLyon-Abla832-WEB-Domains-link_to_shopify

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

**Comment by BrenKt** [comment]
Review from PR  "Connecter un nom de domaine OVHcloud a un hebergement Shopify_JEemlyon" [#6072](https://github.com/ovh/docs/pull/6072) create by @Abla832

File: pages/web_cloud/domains/how_to_connect_domain_to_shopify/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+title: Comment connecter un nom de domaine OVHcloud à un hébergement Shopify
+excerpt: Préparez et configurez la zone DNS de votre nom de domaine OVHcloud pour la connecter à un hébergement Shopify
+updated: 2024-04-03
+---
+
+## Objectif
+
+Si vous possédez un nom de domaine chez OVHcloud et que vous souhaitez le connecter à un hébergement Shopify. Vous trouverez dans ce guide les étapes de préparation et de configuration de votre zone DNS OVHcloud pour permettre la configuration de votre hébergement Shopify.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/how_to_connect_domain_to_shopify/guide.fr-fr.md]:
```suggestion
Vous possédez un nom de domaine chez OVHcloud et vous souhaitez le connecter à un hébergement Shopify. Vous trouverez dans ce guide les étapes de préparation et de configuration de votre zone DNS OVHcloud pour permettre la configuration de votre hébergement Shopify.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_shopify/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+### Configurer vos enregistrements DNS sur votre compte OVHcloud
+
+Connectez-vous à votre [espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr){.external} dans la section `Web Cloud`{.action}. Cliquez sur `Noms de domaine`{.action}, puis choisissez le nom de domaine concerné. Positionnez-vous enfin sur l'onglet `Zone DNS`{.action}.
+
+Le tableau qui s'affiche liste l'ensemble des enregistrements DNS du nom de domaine sélectionné.
+
+![dnszone](images/tab.png){.thumbnail}
+
+Chaque enregistrement DNS peut être modifié en cliquant sur le bouton `...`{.action} à droite de chaque ligne du tableau concernée puis cliquez sur `Modifier l'entrée`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/how_to_connect_domain_to_shopify/guide.fr-fr.md]:
```suggestion
Chaque enregistrement DNS peut être modifié en cliquant sur le bouton `...`{.action} à droite de la ligne du tableau concernée puis en cliquant sur `Modifier l'entrée`{.action}.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_shopify/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+La zone DNS est maintenant configurée pour faire le lien avec un hébergement Shopify.
+
+### Connecter son nom de domaine à Shopify
+
+Les manipulations pour cette étape sont à réaliser depuis l'espace de gestion de Shopify. Nous vous invitons à vous rendre directement à l’étape 2 du guide sur le site de Shopify, en cliquant sur [**ce lien**](https://help.shopify.com/fr/manual/domains/add-a-domain/connecting-domains/connect-domain-manual){.external}.
+
+> [!primary]
+>
+> la vérification de votre nom de domaine peut prendre jusqu’à 48 heures.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/how_to_connect_domain_to_shopify/guide.fr-fr.md]:
```suggestion
> La vérification de votre nom de domaine peut prendre jusqu’à 48 heures.
```

File: pages/web_cloud/domains/how_to_connect_domain_to_shopify/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+Si vous utilisez une offre e-mail OVHcloud ou que vous prévoyez de souscrire à l'une de [nos offres e-mail](https://www.ovhcloud.com/fr/emails/), pour préparer vote zone DNS, consultez notre guide sur la [configuration d’un enregistrement MX](/pages/web_cloud/domains/dns_zone_mx).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/how_to_connect_domain_to_shopify/guide.fr-fr.md]:
```suggestion
Si vous utilisez une offre e-mail OVHcloud ou que vous prévoyez de souscrire à l'une de [nos offres e-mail](https://www.ovhcloud.com/fr/emails/), vous devez également préparer vote zone DNS en conséquence. Consultez notre guide sur la [configuration d’un enregistrement MX](/pages/web_cloud/domains/dns_zone_mx).
```

File: pages/web_cloud/domains/how_to_connect_domain_to_shopify/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
+---
+title: Comment connecter un nom de domaine OVHcloud à un hébergement Shopify
+excerpt: Préparez et configurez la zone DNS de votre nom de domaine OVHcloud pour la connecter à un hébergement Shopify
+updated: 2024-04-03
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/how_to_connect_domain_to_shopify/guide.fr-fr.md]:
```suggestion
updated: 2024-04-09
```

**Pull request merged**